packages/docs/operations-manual/attestation-service.md

```diff
@@ -227,7 +231,9 @@ celocli account:get-metadata $CELO_VALIDATOR_RG_ADDRESS
 
 Attestation Services supports Twilio and Nexmo delivery receipts so that these services can callback to provide delivery information. This triggers retries as needed, even between providers, and enables delivery success metrics to be logged.
 
-Nexmo requires manual configuration to enable this. Go to [Settings](https://dashboard.nexmo.com/settings), and under `Delivery Receipts`, enter the external URL of your Attestation Service appended by `/delivery_status_nexmo` -- for example `http://1.2.3.4:80/delivery_status_nexmo`. This should correspond to the URL printed when Attestation Service is started.
+Nexmo requires manual configuration to enable this. If you have not configured a [Nexmo application](https://dashboard.nexmo.com/applications/), go to [Settings](https://dashboard.nexmo.com/settings), and under `Delivery Receipts`, enter the external URL of your Attestation Service appended by `/delivery_status_nexmo` -- for example `http://1.2.3.4:80/delivery_status_nexmo`. This should correspond to the URL printed when Attestation Service is started.
+
+If you have configured [Nexmo applications](https://dashboard.nexmo.com/applications/), open the matching application, click `Edit`, then enter this value as the `Status URL` (you may also be required to enter an `Inbound URL`, though it will be unused).
```

Comment:
The messages toggle says "Communicate with WhatsApp, Facebook Messenger, MMS and Viber."  The URLs entered here still work for for SMS delivery?

Reply:
yes, it's a bad label name, but appears to be actually the same api.